vermouth/tests/test_repair_graph.py

```diff
@@ -273,7 +288,7 @@ def test_repair_graph_with_mutation_modification(system_mod, resid, mutations, m
     for node_idx in mol:
         if mol.nodes[node_idx].get('resid') == resid:
             if mutations:
-                assert mol.nodes[node_idx]['resname'] == mutations[0]
+                assert mol.nodes[node_idx]['resname'] == mutations[0], mol.nodes[node_idx]
```

Comment:
As far as I can tell this is a completely useless addition. Mol.nodes[node_idx] can be anything even a completely empty dictionary for this statement to assert without error.

Reply:
The second argument to assert gets printed on failure. But indeed it's left over bug-trace output. I'll remoev it